studio/components/layouts/ProjectLayout/LayoutHeader/NotificationsPopover/index.tsx

```diff
@@ -223,7 +223,7 @@ const NotificationsPopover: FC<Props> = () => {
         }
       >
         <Tooltip.Root delayDuration={0}>
-          <Tooltip.Trigger>
+          <Tooltip.Trigger asChild>
```

Comment:
This fixed the error, wasn't 100% sure about this one though (otherwise it was <button> inside <button>